### PR TITLE
add: `topgrade-musl-bin`

### PR DESCRIPTION
**Pull request by derphilipp**
topgrade-musl-bin added, as the latest gnu variant broke is now broken on currently stable debian
As a user you then have the choice to use topgrade-bin or topgrade-musl-bin

### CODE REVIEW

File: packages/topgrade-musl-bin/topgrade-musl-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name="topgrade-musl-bin"
+pkgname="topgrade"
+version="10.2.2"
+url="https://github.com/topgrade-rs/topgrade/releases/download/v${version}/topgrade-v${version}-x86_64-unknown-linux-musl.tar.gz"
+description="Upgrades everything on your system"
+hash="7ad20d8475671b25f8f22ff2c5169a85f4557b114c616f5997e60860f400af7c"
+breaks="${pkgname} ${pkgname}-deb ${pkgname}-app ${pkgname}-git ${pkgname}-bin"
+gives="${pkgname}"
+repology=("project: ${pkgname}")
+maintainer="charitarthchugh <37895518+charitarthchugh@users.noreply.github.com>"
```
Comment (wizard-28) [inline packages/topgrade-musl-bin/topgrade-musl-bin.pacscript:10]: @charitarthchugh, are you willing to maintain this pacscript?

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Comment by charitarthchugh** [comment]
Unfortunately, I cannot

File: packages/topgrade-musl-bin/topgrade-musl-bin.pacscript (hunk 1 above)
Comment (wizard-28) [reply packages/topgrade-musl-bin/topgrade-musl-bin.pacscript:10]: @derphilipp Are you willing to maintain this pacscript?

**Comment by derphilipp** [comment]
Unfortunately not;

File: packages/topgrade-musl-bin/topgrade-musl-bin.pacscript [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+prepare() {
+  true
+}
+
+build() {
+  true
+}
```
Comment (Henryws) [inline packages/topgrade-musl-bin/topgrade-musl-bin.pacscript:17]:
```suggestion
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]
Works.

**Pull request merged**